### ISSUE TITLE
VirtualScroll broke the Angular 4 change dectector

### ISSUE THREAD

**Issue by N4vros** [driftyco/ionic#11151]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[ ] **2.x**
[x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
I updated my project from Ionic 2.2 to Ionic 3.0.1.
Since this update, Angular not refresh the views correctly when i use a VirtualScroll list in my project.
The VirtualScroll list work but when i go to an other page after rendered the list Angular do not detect objects changes. I have to use changeDetectorRef or ngZone in order to fix the problem but it's not a normal two-way data binding behavior.

**Expected behavior:**
Without the bug Angular update the view if i change a value.

**Steps to reproduce:**
I created a Plunker to reproduce the error : http://plnkr.co/edit/UGt11N?p=preview
You need to click on a list item, a new page is pushed and if you click on the button "click here" nothing appear (it's the bug, angular not refresh the view).

**Related code:**

```
<button ion-button (click)="setWork()">Click here</button>
<div *ngIf="work">It's work</div>
```

```
work: boolean;

constructor(public navCtrl: NavController) {
	this.work = false;
}

setWork() {
	this.work = true;
	// work only if we use changeDetectorRef or ngZone
}
```

Thanks for your help.

**Comment by szerner** [comment]
I have that same issues with my app: first page with a VirtualScroll list, clicking on list item pushes a detail page on the navigation stack. There updates only get rendered shortly before returning to the first page (using the back button).

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us. We will look into this.

**Comment by lwbdev** [comment]
Hi, just wanted to echo that I've observed the same issue in an app I'm building. Change detection no longer works after a page is pushed onto the navigation stack after clicking an item in the virtual scroll.

**Comment by jgw96** [comment]
Hello all! This has since been fixed in 3.1.0. Thanks for using Ionic!
